### ISSUE TITLE
 Update Project Profile: Expunge Assist with new description, add tools, wiki and overview links

### ISSUE THREAD

**Issue by Thinking-Panda** [hackforla/website#7379]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [x] In your IDE, open the `_projects/expunge-assist.md` file.
- [x] Observe the existing syntax of the front matter block [^1] in the file.
- [x] Add the following lines of code after `location` variable and its data
   ```
   tools: 
     - Figma
     - Miro
     - Zoom
     - Google Suite
     - FigJam
     - LettuceMeet
   ```
- [x] Find the `description` variable and change the following:
   **From:**
   ```
   description: 'Expunge Assist aims to help people in California with criminal records accomplish record clearance, expungement or reduction and subsequently a second chance as a part of society.
   <br /><br />
   We work with verifiable non-profits, the government and partners to build digital tools that can affect changes in the lives of these justice impacted individuals.'
   ```
   **To:**
   ```
   description: 'Expunge Assist is a free digital tool designed to aid Californians with arrest and conviction histories in clearing their criminal records. A personal statement is the most important proof of rehabilitation filed with an expungement petition. Our letter generator helps people write their statements to the court. The tool does not rely on AI, protects personal privacy, and does not store sensitive information. Community organizers, government agencies, and legal advocates in the clean slate and criminal justice fields are making an incredible impact. Expunge Assist aims to support that work.'
   ```
- [x] Download the PDF from here: https://github.com/hackforla/product-management/blob/master/project-one-sheets/Expunge-Assist-One-Sheet.pdf
- [x] Upload the PDF to this directory: `assets/pdfs`
- [x] Under the `links` variable, add this line of code before the other links:
   ```
     - name: Overview
        url: '../assets/pdfs/Expunge-Assist-One-Sheet.pdf'
   ```
- [x] In the links section, after the GitHub link, add this line of code:
   ```
     - name: Wiki
       url:  'https://github.com/hackforla/expunge-assist/wiki'
   ```
- [x] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [x] Expunge Assist page [^2]
  - [x] the project's card on the Projects' page [^3]  
- [x] Once your pull request is merged, go to #6579 and
  - [x] Check off this issue under the _Dependency_ section
  - [x] If all the issues in the _Dependency_ section are checked off, move #6579] to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
- This issue was initiated by #6579
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/expunge-assist
[^3]: Projects page URL: https://www.hackforla.org/projects/

**Comment by HackforLABot** [comment]
Hi @FamousHero, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)
